README.md

```diff
@@ -41,6 +44,99 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
+
+### Using the released version
+If you want to use the released version of SDM CAP plugin follow the below steps:
+
+1. Delete the sdm and sdm-root folders from your local .m2 repository
+
+2. Clone the sdm repository:
+
+```sh
+   git clone https://github.com/cap-java/sdm
+```
+
+3. Checkout to the branch **deploy**:
+
+```sh
+   git checkout deploy
+```
+
+4. Navigate to the demoapp folder:
+
+```sh
+   cd cap-notebook/demoapp
+```
+
+5. Modify the [xsappname](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/xs-security.json#L1) value in the **xs-security.json**
+
+6. Modify the [xsappname](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L82), [REPOSITORY_ID](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L21) and SDM instance name in the [srv module](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L31) and the [resources section](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L98) values in the **mta.yaml**
+
+7. Build the application:
+
+```sh
+   mbt build
+```
+8. Log in to Cloud Foundry space:
+
+```sh
+   cf login -a <CF-API> -o <ORG-NAME> -s <SPACE-NAME>
+```
+9. Deploy the application:
+
+```sh
+   cf deploy mta_archives/*.mtar
+```
+
+### Using the development version
+If you want to use the development version of SDM CAP plugin follow the below steps:
+
+1. Clone the sdm repository:
+
+```sh
+   git clone https://github.com/cap-java/sdm
+```
+2. Install the plugin in the root folder:
+
+```sh
+   mvn clean install
+```
+
+3. Checkout to the branch **deploy**:
+
+```sh
+   git checkout deploy
+```
+
+4. Navigate to the demoapp folder:
+
+```sh
+   cd cap-notebook/demoapp
+```
+
+5. Modify the [xsappname](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/xs-security.json#L1) value in the **xs-security.json**
+
+6. Modify the [xsappname](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L82), [REPOSITORY_ID](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L21) and SDM instance name in the [srv module](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L31) and the [resources section](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L98) values in the **mta.yaml**
+
+7. Build the application:
+
+```sh
+   mbt build
+```
+8. Log in to Cloud Foundry space:
+
+```sh
+   cf login -a <CF-API> -o <ORG-NAME> -s <SPACE-NAME>
+```
+9. Deploy the application:
+
+```sh
+   cf deploy mta_archives/*.mtar
+```
+
 ## Deploying and testing the application
```

Comment:
Move the section ## Use @cap-java/sdm plugin before deploying and testing section.

Reply:
Add point 2 from "Deployment and testing" to the "Use sdm plugin" section. Please refer to Nodejs readme.
https://github.com/cap-js/sdm?tab=readme-ov-file#use-cap-jssdm-plugin

Reply:
What about the GIF images to show features?

Reply:
Updated file size and added GIFs